influence_benchmark/RL/base_iteration.py

```diff
@@ -98,6 +110,7 @@ def launch(self):
                 self._run_iteration(iteration_step, self.override_initial_traj_path)
             else:
                 self._run_iteration(iteration_step)
+        print(f"Total agent calls: {self.total_agent_calls}")
```

Comment:
I think this prints both the environment and agent calls  (but not PM and TM) if they have the same backend?

Reply:
Yeah that's true, good catch – was not the case when I was testing it, but is now after some later modifications